### PR TITLE
fixed README and index.rst for Python documentation

### PR DESCRIPTION
**Pull request by cgoldberg**
This PR fixes issues with the python README and index.rst docs.

issues fixed that were reported in #1137:
- fixed broken table that was making the document invalid reStructuredText.  In its current state, rendering of the document on PyPI is broken. (it shows raw source rather than rendered reStructuredText)  
  - see: https://pypi.python.org/pypi/selenium/2.48.0
  - this is a regression introduced by commit: https://github.com/SeleniumHQ/selenium/commit/3fc2db275149748b21ea0370d3d3ace2df7c2676
- changed supported Python versions to be more generic:
  - replaced "Python 2.6, 2.7" with "Python 2.6+"
  - replaced "Python 3.2, 3.3" with "Python 3.2+"

other fixes:
- renamed page from "Selenium Client Driver" to "Selenium WebDriver Client - Python Bindings", and removed redundant description.
- updated server download location and server run command to reflect version 2.48.0 (it wasn't updated since 2.46)
- tweaked link to GitHub for viewing source
- fixed a typo in page: s/uisng/using

Note: `py/docs/source/index.rst` is a copy of `py/README` and should be kept in sync.  This PR updates both docs.

### CODE REVIEW

File: py/README [hunk 1]
```diff
@@ -14,20 +12,20 @@ The `selenium` package is used automate web browser interaction from Python.
...
 
-Several browsers/drivers are supported (Firefox, Chrome, Internet Explorer, PhantomJS), as well as the Remote protocol.
+Several browsers/drivers are supported (Firefox, Chrome, Internet Explorer, PhantomJS, etc), as well as the Remote protocol.
 
 Supported Python Versions
 =========================
 
-* Python 2.6, 2.7
-* Python 3.2, 3.3
+* Python 2.6+
```
Comment (bittner) [inline py/README]: I think "2.6, 2.7" is fine. There will be no 2.8 ever according to our BDFL.

File: py/README [hunk 2]
```diff
@@ -14,20 +12,20 @@ The `selenium` package is used automate web browser interaction from Python.
...
+* Python 3.2+
```
Comment (bittner) [inline py/README]: I think "3.2, 3.3, 3.4" is more accurate.

(Has it already been tested with 3.5? Will it automagically be with Python 3.6 once that is released?)

File: py/README [hunk 3]
```diff
@@ -14,20 +12,20 @@ The `selenium` package is used automate web browser interaction from Python.
 +-----------+-----------------------------------------------------------------------------------+
 | **Docs**: | `selenium package API <http://selenium.googlecode.com/git/docs/api/py/api.html>`_ |
 +-----------+-----------------------------------------------------------------------------------+
-| **Dev**:  | https://github.com/SeleniumHQ/selenium                                               |
+| **Dev**:  | https://github.com/SeleniumHQ/selenium                                            |
```
Comment (bittner) [inline py/README]: I would recommend switching the whole table markup to the [simple table markup style](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#simple-tables). The grid tables are in general very fragile (read: error prone).

**Comment by ddavison** [comment]
thanks for reviewing, @bittner .  @cgoldberg , your code does need changes as well as a rebase.  I'm not hugely familiar with the python bindings, so i can't speak for the versioning

**Comment by bittner** [comment]
@cgoldberg Regarding the duplicate content, we should read content out from `README` in `index.rst`. See the [behave-django](https://raw.githubusercontent.com/mixxorz/behave-django/master/docs/index.rst) project docs for an example. The `.. include::` directive is described in the [Sphinx docs](http://sphinx-doc.org/rest.html#directives).

**Comment by cgoldberg** [comment]
fucked up my branch and don't feel like fixing it.  deleting this PR

**Pull request closed without merging**